src/js/controller/controller.js

```diff
@@ -408,7 +408,14 @@ Object.assign(Controller.prototype, {
             }
 
             const playReason = _getReason(meta);
-            const startTime = meta ? meta.startTime : null;
+            let startTime;
+            
+            if (meta && meta === Object(meta)) {
```

Comment:
When is `meta` truthy but not an object?

If `item.startime` is set, why would we only set `meta.startTime ` when `meta` is passed into `play` as an object?

Reply:
@robwalch Theres an integration test where we call ```play()``` from the api with ```true``` as an argument in the last unstable build but I didnt see any case where we pass anything but an object in:

```unknown error: Cannot create property 'startTime' on boolean 'true' ```